### DISCUSSION TITLE
Deploy Jupyter Server Container

### DISCUSSION THREAD

**Discussion by EslamAsHhraf** [polypheny/Polypheny-DB#477] [Q&A] [answered]
How to start using notebooks, when i click button start gives me this error and in [tutorial](https://docs.polypheny.com/en/latest/user_interfaces/notebooks/introduction)  doesn't show hoe to solve or clear for me to get start 

![image](https://github.com/polypheny/Polypheny-DB/assets/71986226/6ba934c9-7800-4c37-b4dc-006fc44c2dfa)

**Comment by hennlo** [comment]
@EslamAsHhraf If you run it for the first time you should follow this small setup guide https://docs.polypheny.com/en/latest/getting_started/setup/docker

And then continuig here:
https://docs.polypheny.com/en/latest/tutorials/notebooks


Or did you already run it before? And after restarting the notebooks feature did not work again?
In that case please open a bug with concise steps how to reproduce this issue.

**Reply by EslamAsHhraf** [reply]
@hennlo okay but in setup guide is said "If Docker is installed locally on the same machine as Polypheny, it will automatically detect and connect to this instance." and docker is installed locally actually and I try to solve problem but I can't 

![image](https://github.com/polypheny/Polypheny-DB/assets/71986226/3b74606b-9d9d-4b38-876b-5bbd60f591d7)

and docker is connected successfully but the can't deploy Jupyter server

![image](https://github.com/polypheny/Polypheny-DB/assets/71986226/784af9be-746a-41aa-881f-f2979eb3b936)

Is there are another way to deploy Jupyter server or tips which I can do. I install docker only is there are anything else should I installed?

**Reply by datomo** [reply, marked as answer]
Hey @EslamAsHhraf 
Are you able to deploy an adapter over Docker (like MongoDB or Postgresql)? If not is the polypheny-docker-connector Docker container running.
Additionally, you might try to shutdown Polypheny, delete your `.polypheny` folder, which per default resides in ~/.polypheny and restart.

**Reply by EslamAsHhraf** [reply]
@datomo it works with me thank you
